rbuilder/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.17-bullseye
+FROM golang:1.18.5-bullseye
```

Comment:
seems fine, I might not put the point release if that's a thing that matters, as we would want to get the next release when it's there.